dask/array/tests/test_fft.py

```diff
@@ -27,9 +31,21 @@ def key(k):
     "ihfft",
 ]
 
+all_nd_funcnames = [
+    "fft2",
+    "ifft2",
+    "fftn",
+    "ifftn",
+    "rfft2",
+    "irfft2",
+    "rfftn",
+    "irfftn",
+]
+
 nparr = np.arange(100).reshape(10, 10)
 darr = da.from_array(nparr, chunks=(1, 10))
 darr2 = da.from_array(nparr, chunks=(10, 1))
+darr3 = da.from_array(nparr, chunks=(10, 10))
```

Comment:
Understood.  I'm curious if we should test a case where there is at least one dimension on which we're not performing an FFT.  

This gave me some confidence that everything was running correctly:

```python
@pytest.mark.parametrize('modname', ['numpy.fft', 'scipy.fftpack'])
@pytest.mark.parametrize('funcname', ['fftn', 'ifftn', 'irfftn', 'rfftn'])
@pytest.mark.parametrize('shape,chunks', [((8, 8, 8, 8), (4, 4, 4, 4))])
@pytest.mark.parametrize('axes', [(0,), (1,), (0, 1), (1, 3), (1, 2, 3), (3, 1)])
@pytest.mark.parametrize('norm', [None, 'ortho'])
def test_multi_dimensional(modname, funcname, shape, chunks, axes, norm):
    fft_mod = pytest.importorskip(modname)
    try:
        func = getattr(fft_mod, funcname)
    except AttributeError:
        pytest.skip("`%s` missing function `%s`." % (modname, funcname))
    x = np.arange(np.prod(shape)).reshape(shape)
    y = da.from_array(x, chunks=chunks)
    y = y.rechunk(tuple(shape[i] if i in axes else None  # single chunks
                        for i in range(len(shape))))  # for fft dimensions

    dfunc = fft_wrap(func)

    assert_eq(func(x, axes=axes),
              dfunc(y, axes=axes))
```